bundles/org.dataflowanalysis.analysis.converter/src/org/dataflowanalysis/analysis/converter/PCMConverter.java

```diff
@@ -64,14 +64,14 @@ public DataFlowDiagramAndDictionary pcmToDFD(String modelLocation,String usageMo
                 .useNodeCharacteristicsModel(nodeCharPath).build();
 
         analysis.initializeAnalysis();
-        var flowGraph = analysis.findFlowGraph();
+        var flowGraph = analysis.findFlowGraphs();
         flowGraph.evaluate();
 
         return processPalladio(flowGraph);
     }
 
-    private DataFlowDiagramAndDictionary processPalladio(FlowGraph flowGraph) {
-        for (AbstractPartialFlowGraph aPFG : flowGraph.getPartialFlowGraphs()) {
+    private DataFlowDiagramAndDictionary processPalladio(FlowGraphCollection flowGraphCollection) {
+        for (AbstractTransposedFlowGraph aPFG : flowGraphCollection.getTransposedFlowGraphs()) {
```

Comment:
Abbreviation is still for PartialFlowGraph, should be aTFG

Reply:
Did not occur to me that we used abbreviations for partial/transposed flow graphs in the code. Fixed in 2e91cb003bdc0da6d61d041ceed27354bcc42e5b